src/Rings/MPolyQuo.jl

```diff
@@ -682,7 +682,7 @@ function Base.:(==)(a::MPolyQuoIdeal{T}, b::MPolyQuoIdeal{T}) where T
 end
 
 @doc Markdown.doc"""
-    simplify(f::MPolyQuoRingElem)
+    simplify(f::MPolyQuoRingElem{T}) where {S<:Union{FieldElem, ZZRingElem}, T<:MPolyElem{S}}
```

Comment:
I know we want to avoid having these type annotations in the docstrings. But in the previous version, the docstring was not found. Since this tends to be a trial-and-error-game, I put it here for now. If it works, I advocate for keeping it that way as an exception.